tests/queries/0_stateless/03214_bitslice_argument_evaluation.sql

```diff
@@ -0,0 +1,10 @@
+-- No arguments passed
```

Comment:
Let's add `-- { echo }` in the beginning, so it'll be easier to debug this test in the future